routes/__tests__/petitions.test.js

```diff
@@ -1,9 +1,49 @@
-const app = require("../../app");
-const request = require("supertest");
+const app = require('../../app');
+const request = require('supertest');
+const { runQuery, getConnection } = require('../../lib/sql');
 
-describe("/petitions", () => {
-  test("it is available", async () => {
-    const response = await request(app).get("/petitions");
-    expect(response.status).toBe(200);
-  });
+describe('/petitions', () => {
+    beforeEach(async () => {
+        app.db = await getConnection(
+            `${__dirname}/../../db/node_api_exercise_tests.sqlite`
+        );
+    });
+    test('it is available', async () => {
+        const response = await request(app).get('/petitions');
+        expect(response.status).toBe(200);
+    });
+
+    test('it returns all the petitions', async () => {
+        await runQuery(app.db, 'delete from petitions;');
+        await runQuery(
+            app.db,
+            `
+      insert into petitions (starter_urn, title, body, created_at, updated_at)
+      values ('urn:changeorg:starter:1','Save our oceans','We need that','2020-01-01 00:00:00','2024-01-01 00:00:00')`
```

Comment:
Also in Rails are able to just do `Petition.create(...)` for example, as a next iteration after this is merged, we should talk about using an ORM & create empty models. Then in the tests the users wouldn't need to write SQL.

https://bookshelfjs.org/ is an option?